policybot/config/welcome/istio.io.yaml

```diff
@@ -0,0 +1,26 @@
+name: istio.io
+type: welcome
+repos:
+  - "istio/bottestrepo"
+
+resenddays: 5
```

Comment:
Every 5 days seems a bit low? I guess its not too spammy to comment, but most active contributors don't even submit a PR every 5 days I would guess